go/ops/local_publisher.go

```diff
@@ -53,7 +53,13 @@ func (*LocalPublisher) PublishLog(log Log) {
 	logrus.StandardLogger().WithFields(fields).Log(level, log.Message)
 }
 
-/// logrusLogLevel maps the current Level of the logrus logger into a pf.LogLevel.
+// PublishStats implements Publisher
+func (*LocalPublisher) PublishStats(event StatsEvent) {
+	// TODO: will this ever get called?
```

Comment:
I'm not sure of the answer to this question myself but it feels weird to leave it like this.

Reply:
Just removed the comment. It's definitely not going to get called at this point.